apps/frontend/app/admin/problem/[problemId]/edit/page.tsx

```diff
@@ -211,41 +69,46 @@ export default function Page({ params }: { params: { problemId: string } }) {
             </FormSection>
 
             <FormSection title="Description">
-              {getValues('description') && (
+              {methods.getValues('description') && (
                 <DescriptionForm name="description" />
               )}
             </FormSection>
 
             <div className="flex justify-between">
               <div className="w-[360px]">
                 <FormSection title="Input Description">
-                  {getValues('inputDescription') && (
+                  {methods.getValues('inputDescription') && (
                     <DescriptionForm name="inputDescription" />
                   )}
                 </FormSection>
               </div>
               <div className="w-[360px]">
                 <FormSection title="Output Description">
-                  {getValues('outputDescription') && (
+                  {methods.getValues('outputDescription') && (
                     <DescriptionForm name="outputDescription" />
                   )}
                 </FormSection>
               </div>
             </div>
 
-            {getValues('testcases') && <TestcaseField blockEdit={false} />}
+            {methods.getValues('testcases') && (
+              <TestcaseField blockEdit={false} />
+            )}
 
             <FormSection title="Limit">
               <LimitForm blockEdit={false} />
             </FormSection>
+
             <TemplateField />
+
             <SwitchField
               name="hint"
               title="Hint"
               placeholder="Enter a hint"
               formElement="textarea"
               hasValue={showHint}
```

Comment:
```suggestion
              hasValue={methods.getValues('hint') !== ''}
```

이렇게 수정하면 `showHint` state가 필요없어질 것 같아요! `showSource` state도 마찬가지입니다!